### PR TITLE
Adding a new DocumentDB + Xamarin Getting Started

### PR DESCRIPTION
**Pull request by codemillmatt**
Tagging @mimig1 and @SnehaGunda for a review of next steps

### CODE REVIEW

**Comment by codemillmatt** [comment]
@mimig1 - anything else I need to do now that you've approved the PR in order to get this merged? Thanks!

**Comment by mimig1** [comment]
@kirankumarkolli - Can you approve/merge this PR?

File: samples/xamarin/ToDoItems/ToDoItems.Core/Services/CosmosDBService.cs [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        static readonly string collectionName = "Items";
+
+        static async Task<bool> Initialize()
+        {
+            if (docClient != null)
+                return true;
+
+            try
+            {
+                docClient = new DocumentClient(new Uri(APIKeys.CosmosEndpointUrl), APIKeys.CosmosAuthKey);
```
Comment (kirankumarkolli) [inline samples/xamarin/ToDoItems/ToDoItems.Core/Services/CosmosDBService.cs]: Please error out in-case the defaults are not updated by customer. This caused CRI's in the past.

Comment (codemillmatt) [reply samples/xamarin/ToDoItems/ToDoItems.Core/Services/CosmosDBService.cs]: @kirankumarkolli - [this](https://github.com/codemillmatt/azure-documentdb-dotnet/blob/5d942e6d9fa6f6b0158d4df715053ebfb8bf6c99/samples/xamarin/ToDoItems/ToDoItems.Core/Helpers/APIKeys.cs#L10) will stop the user from compiling until they entered both the auth key & url.

**Comment by codemillmatt** [comment]
@kirankumarkolli - anything else I need to do for the PR? The referencing docs page is live.

File: samples/xamarin/ToDoItems/ToDoItems.Core/Services/CosmosDBService.cs (hunk 1 above)
Comment (kirankumarkolli) [reply samples/xamarin/ToDoItems/ToDoItems.Core/Services/CosmosDBService.cs]: +1. I like it.

**Pull request merged**